### DISCUSSION TITLE
see other people's smoke during a break

### DISCUSSION THREAD

**Discussion by Inconn** [ppy/osu#22951] [Ideas]
it would be cool to be able to see other people's smoke during breaks, obviously it would require some changes but i think it would be pretty cool

**Comment by zoey-on-github** [comment]
When you say "other people's smoke", do you mean during a multiplayer game? Or do you mean something in single player like the old "replay comments" where you can see what smoke other people have drawn in the past during the break?

**Reply by Inconn** [reply]
i mean more in multiplayer, i think it'd be a cool option to have. execution would be difficult but imo it's a cool idea

**Comment by ItsShamed** [comment]
If it is while spectating, in theory you already can.
If it is during gameplay, then meh, I'm pretty sure it's trickier than you might think — unless you know how to get negative ping, or that you are okay with having massive doodles appearing instantly during breaks.
